omod-1.10/src/main/java/org/openmrs/module/attachments/rest/AttachmentResource1_10.java

```diff
@@ -152,4 +174,32 @@ public static void voidEncounterIfEmpty(EncounterService encounterService, Strin
 			encounterService.voidEncounter(encounter, "foo");
 		}
 	}
+	
+	@Override
+	public List<Attachment> getAttachments(Patient patient, Visit visit, Encounter encounter, boolean includeRetired) {
+		
+		List<Obs> obs = Context.getObsService().getObservationsByPerson(patient);
```

Comment:
Look at [this](https://github.com/openmrs/openmrs-core/blob/dee1e8fc0c445f96e7fdfca690fc7235d2f969c3/api/src/main/java/org/openmrs/api/ObsService.java#L231) kind of method.

Reply:
@mks-d Do you mean to use [this](https://github.com/openmrs/openmrs-core/blob/dee1e8fc0c445f96e7fdfca690fc7235d2f969c3/api/src/main/java/org/openmrs/api/ObsService.java#L231) method or create a method resembling to this.